site/static/main.ts

```diff
@@ -2,6 +2,9 @@
 
 declare const BASEURL, hljs;
 
+// IIFE to prevent function declarations from moving into the global scope
```

Comment:
Shouldn't the only declaration file be a single `vl.d.ts` rather than a bunch of declaration files that reference each other?